src/pages/office/cat-be/office.cat-be.page.html

```diff
@@ -189,17 +191,17 @@ <h4 class="fault-heading">ECO</h4>
                           [formGroup]="form"
                           [isDelegatedTest]="pageState.delegatedTest$ | async"
                           [faultComments]="pageState.drivingFaults$ | async"
-                          [shouldRender]="(pageState.displayDrivingFaultComments$ | async) && (pageState.displayDrivingFault$ | async)"
-                          [outcome]="pageState.testOutcome$ | async" faultType="driving" header="Driving faults"
+                          [shouldRender]="((pageState.displayDrivingFaultComments$ | async) && (pageState.displayDrivingFault$ | async)) || (isDelegated && (pageState.drivingFaults$ | async).length > 0 )"
+                          [outcome]="pageState.testOutcome$ | async" faultType="driving" header="Driving faults {{isDelegated ? (pageState.drivingFaultCount$ | async) : null}}"
```

Comment:
Just verifying that when isDelegated is false, then this doesn't resolve to:

Driving faults null

Reply:
the null will not display